src/odemis/driver/xt_client.py

```diff
@@ -790,6 +790,77 @@ def set_use_case(self, state):
             self.server._pyroClaimOwnership()
             return self.server.set_use_case(state)
 
+    def get_mpp_orientation(self):
+        """
+        Get the current multi probe orientation in degrees.
+
+        :return (float): Multi orientation in degrees
+        """
+        with self._proxy_access:
+            self.server._pyroClaimOwnership()
+            return self.server.get_mpp_orientation()
+
+    def get_aperture_index(self):
+        """
+        Get the current aperture index.
+
+        :return (float): Aperture index (range 0 - 14)
```

Comment:
Sounds like a int to me... are you sure it's a float?
Also, if there is a "..._get_info()" function that's because the range is not fixed, right? So maybe just say "typical range: 0 - 14), see aperture_index_info.

Reply:
I think the get_info is also there for ranges which are fixed (or should be).
But adding the typically is indeed an improvement.

Reply:
The aperture value is actually send and received as a float. But changing it to an int is indeed easy/clearer for the user.